test/data/bad/crs_nolongervalidtype.result

```diff
@@ -0,0 +1 @@
+[{"message":"crs is not a valid GeoJSON type","line":1}]
```

Comment:
@perrygeo "type" is the wrong term here. I think it should read something like "An old-style 'crs' member is NOT RECOMMENDED."